tools/annotation_tools/turk_with_s3/create_jobs.py

```diff
@@ -8,49 +8,58 @@
-    aws_secret_access_key=secret_key,
-    region_name="us-east-1",
-    endpoint_url=MTURK_SANDBOX,
-)
-print("I have $" + mturk.get_account_balance()["AvailableBalance"] + " in my Sandbox account")
-
-
-def create_turk_job(xml_file_path: str, tool_num: int, input_csv: str):
-    # Delete HITs
-    # For use in dev only
-    for item in mturk.list_hits()["HITs"]:
-        hit_id = item["HITId"]
-        print("HITId:", hit_id)
-
-        # Get HIT status
-        status = mturk.get_hit(HITId=hit_id)["HIT"]["HITStatus"]
-        print("HITStatus:", status)
-
-        # If HIT is active then set it to expire immediately
-        if status == "Assignable" or status == "Reviewable":
-            response = mturk.update_expiration_for_hit(HITId=hit_id, ExpireAt=datetime(2015, 1, 1))
-
-        # Delete the HIT
-        try:
-            mturk.delete_hit(HITId=hit_id)
-        except:
-            print("Not deleted")
-        else:
-            print("Deleted")
+def create_turk_job(xml_file_path: str, tool_num: int, input_csv: str, job_spec_csv: str, use_sandbox: bool):
+    if use_sandbox:
+        MTURK_SANDBOX = "https://mturk-requester-sandbox.us-east-1.amazonaws.com"
+    else:
+        MTURK_SANDBOX = "https://mturk-requester.us-east-1.amazonaws.com"
+
+    access_key = os.getenv("AWS_ACCESS_KEY_ID")
+    secret_key = os.getenv("AWS_SECRET_ACCESS_KEY")
+    mturk = boto3.client(
+        "mturk",
+        aws_access_key_id=access_key,
+        aws_secret_access_key=secret_key,
+        region_name="us-east-1",
+        endpoint_url=MTURK_SANDBOX,
+    )
+    print("I have $" + mturk.get_account_balance()["AvailableBalance"] + " in my Sandbox account")
+
+    # Delete HITs - For use in dev only
+    # NOTE: remove if not needed
+    if use_sandbox:
+        for item in mturk.list_hits()["HITs"]:
+            hit_id = item["HITId"]
+            print("HITId:", hit_id)
+
+            # Get HIT status
+            status = mturk.get_hit(HITId=hit_id)["HIT"]["HITStatus"]
+            print("HITStatus:", status)
+
+            # If HIT is active then set it to expire immediately
+            if status == "Assignable" or status == "Reviewable":
+                response = mturk.update_expiration_for_hit(HITId=hit_id, ExpireAt=datetime(2015, 1, 1))
+
+            # Delete the HIT
+            try:
+                mturk.delete_hit(HITId=hit_id)
+            except:
+                print("Not deleted")
+            else:
+                print("Deleted")
 
     # XML file containing ExternalQuestion object.
     # See MTurk API docs for constraints.
     with open(xml_file_path, "r") as fd:
         question = fd.read()
 
     # Where we will save the turk job parameters
-    turk_jobs_df = pd.DataFrame()
+    # if there are existing jobs data, we will load those
+    if os.path.exists(job_spec_csv):
+        turk_jobs_df = pd.read_csv(job_spec_csv)
+    else:
+        turk_jobs_df = pd.DataFrame()
+
     # TODO: make this command line arg
```

Comment:
nit: remove the comment ?

Reply:
another nit: in the comment, can we just add an example line from csv ?
Not sure what the format of the data is (to really understand the if block)